features/multi-column.yml

```
@@ -0,0 +1,20 @@
+name: Multi-column layout
+description: Multi-column layout flows an element's content across one or more columns in a single row, without affecting the `display` property of its children.
```

An idea here, to show a likely entrypoint into this feature:

```suggestion
description: The `columns` property creates a multi-column layout that flows an element's content across one or more columns in a single row, without affecting the `display` property of its children.
```

Also, can you tell me a little more about the "without affecting" clause? I get the impression that this supposed to be making a comparison to something else, but I'm not sure what.

---

It's a distinction from `flexbox` and `grid` that alter the display of child elements and can be a factor in choosing multi-column over these alternatives

---

I chose not to open with `columns` to maintain some consistency with the flexbox and grid layout systems, but those are not distinct properties, so I'm not strongly tied to that

---

Ah, that makes sense. I think ignoring this comment is fine. 👍